src/hooks/xcm/useXcmBridge.ts

```diff
@@ -357,6 +357,19 @@ export function useXcmBridge(selectedToken?: Ref<ChainAsset>) {
         return;
       }
 
+      // check if recipient account has non-zero native asset. (it cannot be transferred to an account with 0 nonce)
+      if (isValidEvmAddress(toAddress)) {
+        const balWei = await getBalance($web3.value!, toAddress);
+        if (Number(ethers.utils.formatEther(balWei)) === 0) {
+          throw Error('the balance of recipient account should be above zero');
```

Comment:
Please put this into i18n file since the text is repeating

Reply:
no idea how to use i18n in .ts file other than .vue. maybe in later, can tackle this. so refactored as separate variable for repeating words.

Reply:
you can do something like.
```
const { t } = useI18n();

customMsg.value = t('dappStaking.toast.staked', {
            amount: stakeAmount,
            dapp: props.dapp.name,
});
```

Check StakePanel.vue for full implementation

Reply:
Fixed